src/account/Settings.tsx

```diff
@@ -92,6 +95,7 @@ interface StateProps {
   walletConnectEnabled: boolean
   biometryEnabled: boolean
   supportedBiometryType: BIOMETRY_TYPE | null
+  shouldShowRecoveryPhraseInSetting: boolean
```

Comment:
could we rename this to `shouldShowRecoveryPhraseInSettings`? 🙏🏻

Reply:
Yes, I just did it. Thanks!